src/priority_queue.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+
+class Node {
+  constructor(data, priority, next) {
+    this.data = data
+    this.priority = priority
+    this.next = next
+  }
+}
+
+export default class Queue {
+  constructor() {
+    this.thequeue = null
+    this.size = 0
+  }
+
+  enqueue(input, priority) {
+    if (!this.thequeue) {
+      this.thequeue = new Node(input, priority, null)
+      this.size++
+      return
```

Comment:
trying to better my understanding here...I think you are using the ```return``` like ```break``` to exit out of the ```if``` conditional? Bc you are not returning any value? 😄  Check me if I am wrong pls